src/email/templating.rs

```diff
@@ -0,0 +1,19 @@
+use crate::http::structs::AybConfigEmail;
+
+const CLI_CONFIRM_TMPL: &str = "To complete your registration, type\n\tayb client confirm {token}";
+const WEB_CONFIRM_TMPL: &str = "To complete your registration, visit\n\t <{url}>";
+
+pub fn render_confirmation_template(config: &AybConfigEmail, token: &str) -> String {
```

Comment:
What if this lived in email.rs and wasn't a `pub fn`? I suspect it will only ever be used inside of `send_registration_email`, so might not need its own module or public reachability.

Reply:
This module is not public outside the email module, so this function cannot be used outside the email module

Reply:
That makes sense. My feedback was slightly different, which is to consider having this live as a non-public function in the original `email.rs` file --- having two files for two short and related functions seemed unnecessary :)

Reply:
Oh alright, I did it that way because in the future the templating logic will be more complex, and thus having a separate file is useful :p